src/test/java/game/race/CarRaceTest.java

```diff
@@ -39,26 +38,16 @@ void init() {
         assertThatThrownBy(() -> inputView.checkInput("0")).isInstanceOf(IllegalArgumentException.class);
     }
 
-    @ParameterizedTest
-    @CsvSource({"3", "4","5"})
-    void 자동차_수와_맞게_각_자동차_이동_결과를_반환_하는_테스트(int cars) {
-        List<Car> carList = Car.getCars(cars);
-        for (Car car : carList) {
-            System.out.println("move count: " + car.getMoveCount());
-        }
-    }
-
     @ParameterizedTest
     @CsvSource({
-            "3, 5",
-            "2, 4",
-            "3, 3"
+            "5,5",
+            "6,6",
+            "7,7",
+            "8,8",
+            "9,9"
     })
-    void 자동차_수와_시도_횟수에_맞게_타이어_자국_수를_확인_하는_테스트(int cars, int tryCnt) {
-        InputView inputView = new InputView();
-        inputView.setVehicleCnt(cars);
-        inputView.setTryCnt(tryCnt);
-
-        CarRacingManager.start(inputView);
+    void 자동차_움직임_확인_하는_테스트(int number, int moveCount) {
+        car.move(number);
+        assertThat(car.getMoveCount()).isEqualTo(moveCount);
```

Comment:
```suggestion
        MovePolicy forward = new ForwardPolicy(); 
        car.move(forward);
        car.move(forward);
        
        assertThat(car.getMoveCount()).isEqualTo(2);
```

이렇게 무조건 전진하는 전략을 전달한다면 moveCount 전달 없이도 해결할 수 있었을 것 같아요